sdk/src/utils/thumbnail.rs

```diff
@@ -25,6 +25,13 @@ const THUMBNAIL_JPEG_QUALITY: u8 = 80;
 /// returns Result (format, image_bits) if successful, otherwise Error
 #[cfg(feature = "file_io")]
 pub fn make_thumbnail(path: &std::path::Path) -> Result<(String, Vec<u8>)> {
+    #[cfg(all(feature = "sfnt", feature = "add_font_thumbnails"))]
+    if path
+        .extension()
+        .map_or(false, |ext| ext == "otf" || ext == "ttf")
```

Comment:
Could use `get_format_from_extension` here?

Reply:
haha, I meant to as well. Good catch, thanks.